test/suites/backup.sh

```diff
@@ -250,6 +250,51 @@ test_backup_import() {
     lxc start b2
     lxc delete --force b2
   fi
+
+  default_pool="$(lxc profile device get default root pool)"
+  default_path="$(lxc profile device get default root path)"
+
+  # Create new storage pools
+  lxc storage create pool_1 dir
+  lxc storage create pool_2 dir
+
+  # Export created container
+  lxc init testimage b3 -s pool_1
+  lxc export b3 "${LXD_DIR}/c3.tar.gz"
+
+  # Remove container and storage pool
+  lxc rm -f b3
+  lxc storage delete pool_1
+
+  # This should succeed as it will fall back on the default pool
+  lxc import "${LXD_DIR}/c3.tar.gz"
+
+  lxc rm -f b3
+
+  # Remove possible containers attached to the profile
+  lxc rm -f c1 c2 || true
```

Comment:
Where do those come from? They should just be deleted as part of whatever test created them.